app/src/main/java/org/thecommonsproject/android/commonhealth/sampleapp/fragments/CategoryListFragment.kt

```diff
@@ -171,14 +178,21 @@ class CategoryListFragment : Fragment() {
         inflater.inflate(R.layout.category_list_item, parent, false)
     ) {
         val title = itemView.findViewById<TextView>(R.id.title_text_view)
+        val countTextView = itemView.findViewById<TextView>(R.id.count_text_view)
         val actionButton = itemView.findViewById<ImageView>(R.id.action_button)
     }
 
     class ContentAdapter(
-        private val categories: List<DataType>,
+        private val categories: List<DataType.ClinicalResource>,
         private val generateOnClickListener: (DataType) -> View.OnClickListener
     ) : RecyclerView.Adapter<CategoryListItemViewHolder>() {
 
+        var resultCounts: Map<DataType.ClinicalResource, Int>? = null
+        fun updateResoutCounts(newResultCounts: Map<DataType.ClinicalResource, Int>) {
```

Comment:
Is the function name supposed to be `resultsCounts`?

Reply:
Renamed to `resultsCounts`